src/main/java/io/mantisrx/api/tunnel/CrossRegionHandler.java

```diff
@@ -450,6 +456,14 @@ private static String getRegion(String uri) {
                 .toLowerCase();
```

Comment:
Is it allowed to use commas in the URL path? are you expecting an url like this:

mantisapi.netflix.net/regions/us-east-1,us-west-2/api/v1/...

Reply:
yep, `,` is a safe character and yep exactly. I want to allow urls you mentioned.